backup_and_restore/restore-acs.adoc

```diff
@@ -8,10 +9,51 @@ toc::[]
 [role="_abstract"]
 You can restore {product-title} from an existing backup by using the `roxctl` command-line interface (CLI).
 
-You can use the `roxctl` CLI to restore {product-title} by using the `restore` command. You require either an API token or your administrator password to run this command.
+Depending upon your requirements and the data you have backed up. You can restore from the following types of backups:
 
-include::modules/restore-acs-roxctl-api.adoc[leveloffset=+1]
+. Restore Central database from the Central database backup. Use this to recover from a database failure or data corruption event.
+It allows you to restore and recover the Central database to its earlier functional state.
+. Restore Central from the Central deployment backup. Use this if you are migrating Central to another cluster or namespace. This option restores the configurations of your Central installation.
```

Comment:
Same here re: mix of sentence fragments and whole sentences. Maybe a description list?

```
Restore Central database from the Central database backup:: Use this to recover from a database failure or data corruption event.
It allows you to restore and recover the Central database to its earlier functional state.
Restore Central from the Central deployment backup:: Use this if you are migrating Central to another cluster or namespace. This option restores the configurations of your Central installation.
```

Or to keep the numbering, some mix of bold and a colon:

```
. *Restore Central database from the Central database backup:* Use this to recover from a database failure or data corruption event.
It allows you to restore and recover the Central database to its earlier functional state.
. *Restore Central from the Central deployment backup:* Use this if you are migrating Central to another cluster or namespace. This option restores the configurations of your Central installation.
```

Altho the latter is maybe getting a bit non-standard.